### ISSUE TITLE
Downsample data for climate project (hw03)

### ISSUE THREAD

**Issue by facusapienza21** [UCB-stat-159-s23/site#6]
## Problem

Last Spring 2022 a lof of students were having problems running the climate dataset in the Hub. The problem was just a memory issue, which didn't really affect what the students where asked to do in the context of their homework. This homework was the most challenging for the students last semester, so it will be good to remove this difficulty. 

## Solution

I think we can just pre-process the dataset before sending them to the students and just downgrande the scale of the xarray objects. This will affect the spatial resolution of the plots, but for the purposes of the project I don't think it will have major impacts. Also it will be nice to have the code running faster. 

What do you think @fperez ? Also, @isaacdsloan do you remember any other issue regarding this homework? I think the memory issue was the largest bottleneck in this project.

**Comment by fperez** [comment]
Yup, agreed on downsampling to limit memory pressure - that problem is somewhat orthogonal to the key course content, so we don't need to burden them with it.

Please double-check that a clean solution runs without issue (while monitoring memory pressure) to be sure the fix works.

Thanks!!

**Comment by isaacdsloan** [comment]
@facusapienza21, love the solution, this will be a live saver for the students! Sounds like it will tackle the main issue of memory as well as run time for building the plots. For me, the biggest headache was the speed of building the plots. This made it difficult to try different things in a timely manner. Let me know if you need help testing out the speed of building the plots as well as checking for any potential memory issues when loading data.
